components/ParameterForm.vue

```diff
@@ -318,6 +326,10 @@ const submitForm = async () => {
     return;
   }
 
+  if (!appStore.globe.tentativelySelectedCountry && appStore.globeParameter?.id && formData.value) {
+    appStore.globe.tentativelySelectedCountry = formData.value[appStore.globeParameter.id];
+  }
```

Comment:
I think this was mainly about persuading TS that everything was OK...
```suggestion
  if (appStore.globeParameter) {
    appStore.globe.tentativelySelectedCountry = formData.value![appStore.globeParameter.id];
  }
```

Reply:
I'm adding this comment:

>  // If the user has not changed the highlighted country since loading the app, then it will still be null,
  // since we don't want to highlight any particular country until the user chooses one. So now we need
  // to set it to the value being submitted in the form. This triggers the Globe component to focus the country.